build.gradle

```diff
@@ -103,6 +103,9 @@ rat {
             'node_modules/**',
             '.idea/**',
             'README.md',
+            'docker/Dockerfile',
+            'docker/docker-compose.yml',
+            'docker/assets/',
```

Comment:
Right, I'm not sure that the docker compose file belongs in the project.  Its specific to mysql.  I suspect the project remains database agnostic, since you can plugin other JDBC drivers.